### ISSUE TITLE
You can scan any reagent passed down from human meat.

### ISSUE THREAD

**Issue by WJohn** [tgstation/-tg-station#8787]
This includes any unobtainable reagents, such as the new reviver implant nanites that are OP as shit. Anyone who does this effectively becomes unkillable.

Steps to reproduce:
1. Inject chem into human
2. Gib human with gibber
3. Get odysseus mech
4. Scan meat
5. Replicate OP chems

**Comment by Jalleo** [comment]
@xxalpha ask me for more info or WJohn on IRC to do with one of your recent PR's

**Comment by Iamgoofball** [comment]
@Jalleo I had nothing to do with the changes to the scanner.

Literally the only thing I want to not be scannable is Adminordrazine/Nanites and that's it. Everything else should be fair game.

**Comment by Cheridan** [comment]
I don't think this is an issue in general, but rather an issue with one single OP chem.

**Comment by optimumtact** [comment]
Well sometimes it's useful to have unobtainable OP chems that you can create over powered effects with, since the chem system is quite nice.

Perhaps some kind of framework for blacklisted chems? it could get messy fast

**Comment by Jalleo** [comment]
@Iamgoofball there were no changes to the scanner I was asking if there was anything you would of wanted to change to the scanner to balance it out a bit with your new chemical stuff. e.g. make certain chems require more power to make or whatever.

**Comment by Iamgoofball** [comment]
Nope, just ban adminord. And that subtype of it added for ERTs

**Comment by Razharas** [comment]
Odysseus needs a rework in general tbh its not really helpful as a healing mech
